vars/multicloudDeploy.groovy

```diff
@@ -13,23 +12,27 @@ import static com.sap.piper.Prerequisites.checkScript
 @Field String STEP_NAME = getClass().getName()
 
 @Field Set GENERAL_CONFIG_KEYS = [
-    /** Defines the targets to deploy on cloudFoundry.*/
+    /** Defines the targets to deploy on Cloud Foundry.*/
     'cfTargets',
     /** Defines the targets to deploy on neo.*/
-    'neoTargets'
+    'neoTargets',
+    /** Executes the deployments in parallel.*/
+    'parallelExecution'
 ]
 
 @Field Set STEP_CONFIG_KEYS = []
 
 @Field Set PARAMETER_KEYS = GENERAL_CONFIG_KEYS.plus([
     /** Defines the deployment type.*/
     'enableZeroDowntimeDeployment',
-    /** Executes the deployments in parallel.*/
-    'parallelExecution',
     /** The source file to deploy to SAP Cloud Platform.*/
-    'source'
+    'source',
+    /** Defines Cloud Foundry service instances to create as part of the deployment.*/
```

Comment:
You should maybe also explain which keys are supported in each instance of the list.

Reply:
How should a user know that he can define apiEndpoint, credentialsId, serviceManifest, manifestVariablesFiles,org,space ...